installation/kubernetes.md

```diff
@@ -22,22 +22,108 @@ LangStream container images are available on the [Github packages registry](http
 * [datastax/langstream-api-gateway](https://github.com/LangStream/langstream/pkgs/container/langstream-api-gateway)
 * [datastax/langstream-cli](https://github.com/LangStream/langstream/pkgs/container/langstream-cli)
 
-### Quick Start
+### Quickstart
 
-To create a LangStream control plane, you will need [kubectl](https://kubernetes.io/docs/reference/kubectl/), [helm cli](https://helm.sh/docs/intro/install/), and a running K8s cluster.
+To create a LangStream control plane, you will need [kubectl](https://kubernetes.io/docs/reference/kubectl/), [helm cli](https://helm.sh/docs/intro/install/), and a running K8s cluster with a recent version.
 
+Connect to your GCP cluster:
+```bash
+gcloud container clusters get-credentials <langstream-cluster> --region us-east1 --project gcp-techpubs
+Fetching cluster endpoint and auth data.
+Kubernetes v1.21.0-alpha+eabdbc0a40fe7efda92e10270f27b0a3485fb743
+kubeconfig entry generated for langstream-cluster.
+```
+
+Connect to your Azure cluster:
+```bash
+az account set --subscription 249a85c0-95ce-41be-8ede-149bce428ae5
+az aks get-credentials --resource-group k8s-resource-group --name dev
+Merged "dev" as current context in /Users/mendon.kissling/.kube/config
+```
+
+Add the LangStream chart repo to your Helm installation and update it to the latest version:
 ```bash
-kubectl apply -f https://raw.githubusercontent.com/LangStream/langstream/main/helm/examples/minio-dev.yaml
 helm repo add langstream https://langstream.ai/charts
-helm repo update
+helm repo update langstream
+```
+
+Modify the values.yaml file you'll be deploying with to configure the external codeStorage component.
+This component stores the state of LangStream applications in an S3 API-compatible bucket.
+These values can be found in your storage provider's dashboard.
+
+Azure:
+```
+codeStorage:
+  type: azure
+  configuration:
+    endpoint: https://<storage-account>.blob.core.windows.net
+    container: langstream
+    storage-account-name: <storage-account>
+    storage-account-key: <storage-account-key>
+```
+
+S3:
+```
+codeStorage:
+  type: s3
+  configuration:
+    access-key: <aws-access-key>
```

Comment:
we need to add the bucket name here

also I would add something for users on GKE

We can add a reference to the MinIO website in case you want to configure an internal S3 compatible storage service